packages/cli-platform-ios/src/commands/logIOS/index.ts

```diff
@@ -15,7 +15,11 @@ import {Device} from '../../types';
 function findAvailableDevice(devices: {[index: string]: Array<Device>}) {
   for (const key of Object.keys(devices)) {
     for (const device of devices[key]) {
-      if (device.availability === '(available)' && device.state === 'Booted') {
+      if (
+        (device.availability === '(available)' ||
+          device.isAvailable === true) &&
+        device.state === 'Booted'
+      ) {
```

Comment:
Question - is there a chance that device is "Booted" and not available? 🤔 
If not then I think this logic could be simplified a bit and we won't care anymore about `availability` v `isAvailable` mismatch :)

Reply:
I think you're probably right in this case. Will have to double-check, but it doesn't really make sense to be unavailable but Booted for a simulator, but similar logic was there before. Maybe there is an edge case 🤷